docs/proposals/kubelet-eviction.md

```diff
@@ -0,0 +1,201 @@
+# Kubelet - Eviction Policy
+
+**Author**: Derek Carr (@derekwaynecarr)
+
+**Status**: Proposed
+
+This document presents a specification for how the `kubelet` evicts pods when compute resources are too low.
+
+## Motivation
+
+The node needs a mechanism to preserve stability when available compute resources are low.
+
+## Scope of proposal
+
+This proposal defines a pod eviction policy for reclaiming compute resources.
+
+In the first iteration, it focuses on memory; later iterations are expected to cover
+other resources like disk.  The proposal focuses on a simple default core policy
+intended to cover the broadest class of user workloads.
+
```

Comment:
Can we clarify the higher level requirements or goals explicitly before proposing solutions?